build.rs

```diff
@@ -95,7 +95,11 @@ fn reformat_file(path: &std::path::Path) {
         .expect("failed to open file");
     let mut data = String::new();
     f.read_to_string(&mut data).expect("failed to read file.");
-    data = data.replace("\n\n", "\n")
+    data = data
+        .replace("\r\n\r\n", "\r\n")
+        .replace("\r\n\r\n", "\r\n")
+        .replace("\r\n\r\n", "\r\n")
+        .replace("\n\n", "\n")
```

Comment:
What is this for?

Reply:
changes in rustfmt (nightly vs stable) change line ending char sequence it seems. Had to add it to account for both. technically we can have an ugly definitions.rs checked in but I'd rather have it formatted to be able to check the code there.